packages/core/admin/index.js

```diff
@@ -48,15 +48,15 @@ function getCustomWebpackConfig(dir, config) {
   return webpackConfig;
 }
 
-async function build({ plugins, dir, env, options, optimize, forceBuild }) {
-  const buildAdmin = await shouldBuildAdmin({ dir, plugins });
+async function build({ plugins, dir, env, options, optimize, forceBuild, useTypeScript }) {
```

Comment:
@Convly you'll need to provide the @strapi/admin package with this variable.

Reply:
Thanks for the notice :slightly_smiling_face: